module02/solution.py

```diff
@@ -0,0 +1,75 @@
+IS_TRUE = True
+IS_FALSE = False
+
+def num_add(a, b):
+    """Return (a + b)."""
+    return a + b
+
+def num_sub(a, b):
+    """Return (a - b)."""
+    return a - b
+
+def num_mul(a, b):
+    """Return (a * b)."""
+    return a * b
+
+def num_div(a, b):
+    """Return (a / b)."""
+    return a / b
+
+def num_floor(a, b):
+    """Return floor (a / b)."""
+    return a // b
+
+def num_rem(a, b):
+    """Return remainder of (a / b)."""
+    return a % b
+
+PANCAKE_INGREDIENTS = {
+    'flour' : 2,
+    'eggs' : 4,
+    'milk' : 200,
+    'butter' : False,
+    'salt' : 0.001
+    }
+
+def ingredient_exists(ingr, dict):
+    """Check if an ingredient exists."""
+    if dict.get(ingr):
```

Comment:
This IF statement is a bit redundant if you think about it a bit more.

Reply:
This would also wrongly return False if we have key but its value is None / False.

Reply:
Tried to resolve this in second commit.